transforms/__testfixtures__/update-react-imports/variable-already-used.input.js

```diff
@@ -0,0 +1,7 @@
+import * as React from "react";
+
+React.createElement('div', {});
+
+createElement('someFunction');
```

Comment:
This hasn't been manually destructed so would likely fail in runtime, (or rather, the assumption will be that it's defined globally) Could you explain this testcase?